<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassBPage</class>
 <widget class="QWidget" name="ClassBPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClassBPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_manual">
     <property name="text">
      <string> Ручной режим</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox_numberClassB"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_Active">
     <property name="text">
      <string>Активна</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_addClassB">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>188</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="5">
    <widget class="ToolBox" name="widgetSimulatorB" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolBox</class>
   <extends>QWidget</extends>
   <header>ToolBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
